src/autocomplete/DuckDuckGoProvider.js

```diff
@@ -1,4 +1,21 @@
+/*
```

Comment:
Hmm, so of https://github.com/matrix-org/matrix-react-sdk/tree/develop/src/autocomplete, the 'Fix a bunch of linting errors' and 'Bulk change counterpart imports' are me, so everything apart from Autocompleter.js and Components.js need Vector 2017 headers.